libs/common-services/src/test/java/uk/gov/di/ipv/core/library/service/ConfigServiceTest.java

```diff
@@ -381,8 +382,16 @@ private TestConfiguration getTestConfiguration(String testDataSet, String featur
     void getCriPrivateApiKeyForActiveConnectionShouldReturnApiKeySecret() {
         environmentVariables.set("ENVIRONMENT", "test");
         Map<String, String> apiKeySecret = Map.of("apiKey", "api-key-value");
+
+        String json =
+                assertDoesNotThrow(
```

Comment:
(Do we need `assertDoesNotThrow` - could we just let the exception fail the test?)

Reply:
@Joe-Edwards-GDS Yes I guess, I figured it was more in line with the testing framework, but happy either way